tools/runScheduledTasks.php

```diff
@@ -19,5 +19,7 @@ class runScheduledTasks extends \PKP\cliTool\ScheduledTaskTool
 {
 }
 
+// app('pkpJobQueue')->runJobsInQueue();
```

Comment:
May as well remove this, since it's handled elsewhere!

Reply:
Done.